api/v1/notebook_types.go

```diff
@@ -62,7 +62,7 @@ type NotebookStatus struct {
 	// Conditions is the list of conditions that describe the current state of the Notebook.
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
-	// Upload contains details the controller returns from a requested signed upload URL.
+	// Image contains the status of the image. Upload URL is reported here.
 	Image ImageStatus `json:"upload,omitempty"`
```

Comment:
Shouldn't this match the others?

```suggestion
	Image ImageStatus `json:"image,omitempty"`
```